applications/Chat/coati/models/base/critic.py

```diff
@@ -44,9 +44,8 @@ def forward(self,
 
         if action_mask is not None and self.use_action_mask:
```

Comment:
Using action mask here means to only use state (prompt) to compute value, as described in RL patterns.

Reply:
Sorry for my misunderstanding. So here uses one step MDP if I understand this correctly? Maybe we should change to multi step MDP and compute values for every answer input.